packages/forms-web-app/src/controllers/examination/check-submission-item/utils/get-back-link-url.js

```diff
@@ -0,0 +1,55 @@
+const {
+	getSubmissionItemType,
+	getSubmissionItemPersonalInformation,
+	getActiveSubmissionItemFiles,
+	getActiveSubmissionItem
+} = require('../../session/submission-items-session');
+const {
+	routesConfig: {
+		examination: {
+			directory,
+			pages: {
+				evidenceOrComment,
+				personalInformation,
+				personalInformationComment,
+				personalInformationCommentFiles,
+				personalInformationFiles,
+				personalInformationWhichCommentFiles,
+				personalInformationWhichFiles
+			}
+		}
+	}
+} = require('../../../../routes/config');
+
+const getBackLinkUrl = (session) => {
+	const activeSubmissionItem = getActiveSubmissionItem(session);
+	const submissionItemType = getSubmissionItemType(activeSubmissionItem);
```

Comment:
this make sense based on the conversations we have had. 

Just looks a but heavy